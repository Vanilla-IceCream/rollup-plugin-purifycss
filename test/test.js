import { rollup } from 'rollup';

import purifycss from '../src';

process.chdir(__dirname);

describe('rollup-plugin-purifycss', () => {
  it('should handle', async () => {
    const { generate } = await rollup({
      input: './fixtures/basic/main.js',
      plugins: [
        purifycss({
          content: ['./fixtures/basic/main.js', './fixtures/basic/index.html'],
          options: {
            info: true,
          },
        }),
      ],
    });

    const res = await generate({ format: 'iife', name: 'purifycss' });

    expect(res.output[0].code).toMatchSnapshot();
  });
});
