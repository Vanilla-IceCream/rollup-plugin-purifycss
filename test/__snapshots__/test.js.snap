// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rollup-plugin-purifycss should handle 1`] = `
"(function () {
  'use strict';



}());
"
`;
